hnn_core/tests/test_network.py

```diff
@@ -28,3 +28,11 @@ def test_network():
     for ev_input in params['t_ev*']:
         type_key = ev_input[2: -2] + ev_input[-1]
         assert len(net.gid_dict[type_key]) == net.n_cells
+
+    # Test read/write spikes consistency
```

Comment:
I think you need to add a function called `test_spikes` and check each of the methods. Here you haven't tested the `__repr__` for instance. See [how we test repr in MNE](https://github.com/mne-tools/mne-python/blob/ddc007f44c5eb840fcfd16dd3987b6535bed9f44/mne/forward/tests/test_forward.py#L55-L59)